menu/additional_apps.sh

```diff
@@ -32,11 +33,16 @@ choice=$(whiptail --title "Which apps do you want to install?" --checklist "Auto
 "SMB-mount" "(Connect to SMB-shares from your local network)" OFF 3>&1 1>&2 2>&3)
 
 case "$choice" in
-    *"Fail2ban"*)
+    *"Fail2ban "*)
         clear
         print_text_in_color "$ICyan" "Downloading Fail2ban.sh..."
         run_script APP fail2ban
     ;;&
+    *"Fail2ban-Statuscheck"*)
+        clear
```

Comment:
Please add a check if Fail2ban is installed.

Thanks!

Reply:
done 👍